src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$dark: #060b35;
+$blue: #00acdc;
+$gray: #616070;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+@mixin star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+@mixin button-buy {
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $gray;
+    margin-top: 4px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $dark;
+  }
+
+  @for $i from 1 through 5 {
+    &__star--#{$i} {
+      background-image: url('../images/star.svg');
+
+      @include star;
+
+      &--filled {
+        background-image: url('../images/star-active.svg');
+
+        @include star;
+      }
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-stars {
+    display: flex;
+  }
+
+  &__reviews-total {
+    font-size: 10px;
+    color: $dark;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &_title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $gray;
+    }
+
+    &_total {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $dark;
+    }
+  }
+
+  &__buy-button {
+    @include button-buy;
+
+    background-color: $blue;
+    border: none;
+    color: $white;
+
+    &:hover {
+      @include button-buy;
+
+      background-color: $white;
+      border: 1px solid $blue;
+      color: $blue;
+    }
```

Comment:
There's no need to include `@include button-buy;` inside the `:hover` state as it repeats the same properties already defined. This is redundant and could be removed to keep your code DRY.